lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -9,14 +11,23 @@ module.exports = {
 
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
 
+    const coreCloudFormationTemplate = this.serverless.utils.readFileSync(
```

Comment:
Can you move this into a separate method so the code isn't here twice in the same file? Or is that not possible because of promises?